web/static/ts/utils/dom.ts

```diff
@@ -29,3 +29,28 @@ export function insertAfter(elem: Element, refElem: Element): Element {
 
     return refElem.parentElement.insertBefore(elem, refElem.nextElementSibling);
 }
+
+/**
+ * @param alert The alert caused by the User/System.
+ */
```

Comment:
If you're going to bother adding docstrings you may as well go the full distance - describe what the function does too.